plugins/woocommerce/src/Blocks/BlockTypes/ProductDetails.php

```diff
@@ -191,7 +208,8 @@ private function create_accordion_item_block( $title, $content ) {
 	 * @return array Parsed block.
 	 */
 	private function inject_parsed_accordion_blocks( $parsed_block, $accordion_blocks ) {
-		if ( 'woocommerce/accordion-group' === $parsed_block['blockName'] ) {
+		$accordion_group_name = Utils::wp_version_compare( '6.9', '>=' ) ? 'core/accordion' : 'woocommerce/accordion-group';
+		if ( $accordion_group_name === $parsed_block['blockName'] ) {
```

Comment:
I would prefer if in this `if`, we check for both blocks, no matter the version of WordPress. That would cover the case of somebody having added the block in WP 6.8 and then upgrading to WP 6.9. The old accordions would still be used, even if the store is in WP 6.9.

```suggestion
		$accordion_group_name = Utils::wp_version_compare( '6.9', '>=' ) ? 'core/accordion' : 'woocommerce/accordion-group';
		if ( 'core/accordion' === $parsed_block['blockName'] || 'woocommerce/accordion-group' === $parsed_block['blockName'] ) {
```